{
	"id": "CVE-2020-26053",
	"sourceIdentifier": "cve@mitre.org",
	"vulnStatus": "Rejected",
	"published": "2020-09-30T18:15:26.677",
	"lastModified": "2023-11-07T03:20:28.813",
	"descriptions": [
		{
			"lang": "en",
			"value": "Rejected reason: DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Reason: This candidate was withdrawn by its CNA. Further investigation showed that it was not a security issue. Notes: none"
		}
	],
	"references": [],
	"metrics": {}
}
